src/index.css

```diff
@@ -1,22 +1,64 @@
 body {
-  font: 14px "Century Gothic", Futura, sans-serif;
-}
-
-ol,
-ul {
-  padding-left: 30px;
-  color: #0c5460;
-  background-color: #d1ecf1;
-  border-color: #bee5eb;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
+    Ubuntu, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  color: #838689;
 }
 
 label {
   margin: 6px;
+  margin-bottom: 10px;
+  font-size: 12px;
+  display: flex;
+  font-weight: 600;
+  align-items: center;
+  justify-content: center;
 }
 
 input {
-  width: 70px;
+  width: 90px;
   margin-left: 6px;
+  border-radius: 5px;
+  color: #838689;
+  font-weight: 600;
+  font-size: 12px;
+  display: block;
+  padding: 10px 18px 11px;
+  box-shadow: 0 1px 3px 1px #838689;
+  text-size-adjust: 100%;
+  transition: box-shadow 0.3s ease-in-out;
+  border: none;
+  background-color: #f6f7fb;
+  border: solid 1px #ccc;
+}
+
+input:hover {
+  box-shadow: 0 3px 6px 1px #838689;
+}
+
+input[type="text"]:focus {
+  outline: none;
+  border-color: transparent;
+  box-shadow: 0 0 0 2px #43d1af;
+}
+
+.button {
+  background: #ffffff;
+  cursor: pointer;
+  color: #43d1af;
+  border-radius: 50%;
+  border: #ffffff;
+  overflow: hidden;
+  margin: 10px;
+  transition: all 150ms linear;
```

Comment:
You use `ms` here, but below you use `s`, may want to pick one or the other for consistency.